$NetBSD: patch-skins_classic_print.css,v 1.1 2013/06/15 16:08:09 taca Exp $

Fix from repository:
* commit 50d0a12933f5b58506cfe0df477eeb81b727d41e:
  Add styles for image thumbnails in print mode (#1489134)

--- skins/classic/print.css.orig	2013-05-16 18:06:37.000000000 +0000
+++ skins/classic/print.css
@@ -136,3 +136,31 @@ div.message-part blockquote blockquote b
   border-left: 2px solid #bb0000;
   border-right: 2px solid #bb0000;
 }
+
+p.image-attachment
+{
+  position: relative;
+  padding: 1em;
+  border-top: 1px solid #ccc;
+}
+
+p.image-attachment a.image-link
+{
+  float: left;
+  display: block;
+  margin-right: 2em;
+  min-width: 160px;
+  min-height: 60px;
+  text-align: center;
+}
+
+p.image-attachment .image-filename
+{
+  display: block;
+  line-height: 1.6em;
+}
+
+p.image-attachment .attachment-links
+{
+  display: none;
+}
